hedera-node/hedera-mono-service/src/testFixtures/java/com/hedera/test/utils/KeyUtils.java

```diff
@@ -16,65 +16,68 @@
-                    .addKeys(KEY_BUILDER.apply(A_NAME))
-                    .addKeys(KEY_BUILDER.apply(B_NAME))
-                    .addKeys(KEY_BUILDER.apply(C_NAME)))
+            .keyList(KeyList.newBuilder()
+                    .keys(KEY_BUILDER.apply(A_NAME).build(),
+                            KEY_BUILDER.apply(B_NAME).build(),
+                            KEY_BUILDER.apply(C_NAME).build()))
             .build();
     public static final Key A_COMPLEX_KEY = Key.newBuilder()
-            .setThresholdKey(ThresholdKey.newBuilder()
-                    .setThreshold(2)
-                    .setKeys(KeyList.newBuilder()
-                            .addKeys(KEY_BUILDER.apply(A_NAME))
-                            .addKeys(KEY_BUILDER.apply(B_NAME))
-                            .addKeys(A_THRESHOLD_KEY)))
+            .thresholdKey(ThresholdKey.newBuilder()
+                    .threshold(2)
+                    .keys(KeyList.newBuilder()
+                            .keys(KEY_BUILDER.apply(A_NAME).build(),
+                                    KEY_BUILDER.apply(B_NAME).build(),
+                                    A_THRESHOLD_KEY)))
             .build();
     public static final Key B_COMPLEX_KEY = Key.newBuilder()
-            .setThresholdKey(ThresholdKey.newBuilder()
-                    .setThreshold(2)
-                    .setKeys(KeyList.newBuilder()
-                            .addKeys(KEY_BUILDER.apply(A_NAME))
-                            .addKeys(KEY_BUILDER.apply(B_NAME))
-                            .addKeys(A_COMPLEX_KEY)))
+            .thresholdKey(ThresholdKey.newBuilder()
+                    .threshold(2)
+                    .keys(KeyList.newBuilder()
+                            .keys(KEY_BUILDER.apply(A_NAME).build(),
+                                    KEY_BUILDER.apply(B_NAME).build(),
+                                    A_COMPLEX_KEY)))
             .build();
     public static final Key C_COMPLEX_KEY = Key.newBuilder()
-            .setThresholdKey(ThresholdKey.newBuilder()
-                    .setThreshold(2)
-                    .setKeys(KeyList.newBuilder()
-                            .addKeys(KEY_BUILDER.apply(A_NAME))
-                            .addKeys(KEY_BUILDER.apply(B_NAME))
-                            .addKeys(B_COMPLEX_KEY)))
+            .thresholdKey(ThresholdKey.newBuilder()
+                    .threshold(2)
+                    .keys(KeyList.newBuilder()
+                            .keys(KEY_BUILDER.apply(A_NAME).build(),
+                                    KEY_BUILDER.apply(B_NAME).build(),
+                                    B_COMPLEX_KEY)))
             .build();
 
-    public static List<Key> sanityRestored(List<? extends HederaKey> jKeys) {
+    public static List<com.hederahashgraph.api.proto.java.Key> sanityRestored(List<? extends HederaKey> jKeys) {
         return jKeys.stream()
                 .map(jKey -> {
                     try {
```

Comment:
This try block has a merge error, the throw should be inside the catch, not after it.
Also, I think the return could have remained List<Key> by converting the JKey to PBJ key on line 84.

Reply:
Not sure I understand. This is an existing method and is used in multiple tests in mono-service

Reply:
A lot of them were damaged like this in one of the earlier merges.  The throws on line 87 belongs inside the catch block, otherwise the lambda won't throw (because of the return, otherwise it would throw always).

Reply:
Made the change to move it inside catch